tests/shared/features/shared/email_steps.feature

```diff
@@ -33,6 +33,19 @@ Feature: Test Spreewald's email steps
         '''
       """
 
+    # Test with invalid header
+    Then the step with an unsupported email header should raise:
+      """
+      Then an email should have been sent with:
+        '''
+        From: from@example.com
+        Reply-To: reply-to@example.com
+        Invalid-Header: reply-to@example.com
+        To: to@example.com
+        Subject: SUBJECT
+        '''
+      """
+
```

Comment:
Denke das ist okay, man hätte in dem Fall aber schon auch eine Spec für den mail_finder machen können. Zwingt dich eventuell in der Spec `mail` zu requiren und dann irgendwie `Mail.new` zu machen, aber glaube es wäre möglich gewesen.

So ists aber aber auch gut.

Reply:
Ich hatte es versucht, ich glaube ActiveSupport String Extensions bräuchte man auch. Wenn jetzt nochmal so ein Requirement kommt, mit Fehlerbehandlung oder ähnliches würde ich es nochmal versuchen mit RSpec und den passenden requires zu speccen und dann den cucumber test darüber auch abzubilden.